src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverProjectExtension.kt

```diff
@@ -31,35 +55,36 @@ public interface KoverProjectExtension {
     public fun useJacocoTool()
 
     /**
-     * Kover Coverage Tool with default version [KOVER_TOOL_DEFAULT_VERSION].
+     * Coverage Tool by [JaCoCo](https://www.jacoco.org/jacoco/).
      */
-    public fun useKoverTool(version: String)
+    public fun useJacocoTool(version: String)
 
     /**
-     * Coverage Tool by [JaCoCo](https://www.jacoco.org/jacoco/).
+     * Excludes form report all classes, defined in Java source files.
+     *
+     * As a side effect, reports cease to depend on Java compilation task.
      */
-    public fun useJacocoTool(version: String)
+    public fun excludeJavaCode()
 
     /**
      * Disables instrumentation of specified tests.
      *
      * This means that even if the excluded test is executed, the function calls that occurred in it will not be counted in the reports.
      *
-     * As a side effect, reports cease to depend on the specified test tasks.
+     * As a side effect, Kover reports cease to depend on the specified test tasks.
      */
     public fun excludeTests(config: Action<KoverTestsExclusions>)
 
     /**
      * _Experimental_
      *
-     * Exclude specified compilation units from report.
+     * Exclude specified Kotlin compilations from report.
      *
      * The unit is typical for each type of project, for example, for Kotlin JVM it is Source Set, for Kotlin MPP it is target or Kotlin compilation.
-     * Also, JVM sources are an independent compilation unit.
      *
-     * As a side effect, when any compilation unit is excluded, reports cease to depend on the task of compiling this unit.
+     * As a side effect, when any compilation is excluded, Kover reports cease to depend on the appropriate compilation task.
      */
-    public fun excludeSources(config: Action<KoverSourcesExclusions>)
+    public fun excludeCompilations(config: Action<KoverCompilationsExclusions>)
```

Comment:
This one doesn't have either tests or examples of use.

Should it really be here?

Reply:
This is an experimental API, I don't want to give examples without full-fledged tests.

 I may hide this block until the tests are written.